compiler/rustc_infer/src/infer/error_reporting/note.rs

```diff
@@ -1,421 +0,0 @@
-use crate::errors::{
```

Comment:
Everything in this module was region-related so I just combined it into the new region module lol